### PR TITLE
Add Recipe#5 

### PR DESCRIPTION
**Pull request by Teresakoetter**
"As a User I want to add more recipes to the collection"

Create a POST-Endpoint "/api/recipes"

In the frontend I want to add a Recipe via a formular.

Should be an own page with own routing

write unit- and integrationtests

the user wants to choose the name

the backend should generate a random id

Everyone should be able to do this no "admin"

### CODE REVIEW

File: backend/src/test/java/de/neuefische/gruppe1/backend/RecipeRepoTest.java [hunk 1]
```diff
@@ -33,9 +34,10 @@ void getAll_ShouldReturnFalseWhenEmpty() {
 
         //THEN
         List<Recipe> expected = new ArrayList<>();
-        expected.add(new Recipe("1", "Pizza"));
-        expected.add(new Recipe("2", "Gurkensalat"));
+        expected.add(new Recipe("1", "Erstes Rezept, voll gut!"));
+        expected.add(new Recipe("2","Zweites Rezept, noch besser!"));
```
Comment (abnaemi) [inline backend/src/test/java/de/neuefische/gruppe1/backend/RecipeRepoTest.java:39]: Sieht gut aus, evtl. statt add und dann die Eigenschaften vlt. Objekte hinzufügen die man oben erstellt hat?

File: backend/src/test/java/de/neuefische/gruppe1/backend/RecipeRepoTest.java
Comment (lukas-kraus) [inline backend/src/test/java/de/neuefische/gruppe1/backend/RecipeRepoTest.java:1]: Remove comment

File: frontend/src/App.css [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-  text-align: center;
+    text-align: center;
 }
 
 .App-logo {
-  height: 40vmin;
-  pointer-events: none;
+    height: 40vmin;
+    pointer-events: none;
 }
```
Comment (lukas-kraus) [inline frontend/src/App.css:8]: Where is the logo?!?!?!?!?!??!?!?!

File: frontend/src/App.css [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
-  }
-  to {
-    transform: rotate(360deg);
-  }
+    from {
+        transform: rotate(0deg);
+    }
+    to {
+        transform: rotate(360deg);
+    }
```
Comment (lukas-kraus) [inline frontend/src/App.css:37]: Animation for what?!

File: frontend/src/Header.tsx [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
+            <h1> Hier gibt es bald was zu Futtern</h1>
```
Comment (lukas-kraus) [inline frontend/src/Header.tsx:6]:
```suggestion
            <h1>Hier gibt es bald was zu Futtern</h1>
```

**Review by lukas-kraus — APPROVED** [review]

**Pull request merged**